doc/source/data/api/input_output.rst

```diff
@@ -1,150 +1,149 @@
-.. _data_api:
 
-Ray Datasets API
-================
+Input/Output
```

Comment:
I see that you went with a flat list rather than grouping these APIs by data type, e.g. tabular, tensor, text, binary, etc. Looking at it now, I think that the flat list is better for quick discoverability, but we might want to reexamine such a grouping if we add support for a few more file formats to keep this section's size in the ToC manageable.

But yeah, the flat list looks good to me for now!

Reply:
Yeah, basically we can follow the headers in https://docs.ray.io/en/master/data/creating-datasets.html, let me know if you think it's worthwhile to add in this PR!

Reply:
I think the current flat list looks good to me!